src/lualib/5.0/SourceMapTraceBack.ts

```diff
@@ -0,0 +1,69 @@
+// TODO: In the future, change this to __TS__RegisterFileInfo and provide tstl interface to
```

Comment:
(See Error.ts comment) Maybe use `_Version` in existing SourceMapTraceBack.ts to work around differences. Since it is not performance critical.

Reply:
Implemented with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/581485960ed7c053ba1e2e20a68f4dacb5726629.